tests/virtctl/ssh.go

```diff
@@ -74,12 +45,41 @@ var _ = Describe("[sig-compute][virtctl]SSH", func() {
 			"-t", "-o StrictHostKeyChecking=no",
 			"-t", "-o UserKnownHostsFile=/dev/null",
 			"--command", "true",
-			vmi.Name)
+			vmiName)
 		Expect(err).ToNot(HaveOccurred())
 		cmd.Env = append(cmd.Env,
 			"SSH_AUTH_SOCK=/dev/null")
 		out, err := cmd.CombinedOutput()
 		fmt.Println(string(out))
```

Comment:
Can you assert that this isn't empty instead of printing it out?

Reply:
:+1: